controllers/employeeDashboard.controller.js

```diff
@@ -0,0 +1,61 @@
+const User = require("./../modules/user.module");
+const {
+    setHomeAddress,
+    setVacationRequest,
+    getUserById,
+    getAllUsers,
+} = require("./../DAL/user.DAL");
+const bcrypt = require("bcrypt");
+const _ = require("lodash");
+const {
+    invalidId,
+    missing_homeAddress,
```

Comment:
be consistent with naming. missingHomeAddress is better.
what do invalidId and missing_homeAddress do? give more meaningful names